.github/workflows/continuous-integration.yml

```diff
@@ -50,7 +50,7 @@ jobs:
       run: Get-ChildItem -Recurse -Include *.sln | ForEach-Object {dotnet build $_ --configuration Release --no-restore; if ($LASTEXITCODE -ne 0 ) {throw "Build for $_ FAILED"; }} 
 
     - name: Test .Net
-      run: Get-ChildItem -Recurse -Include *.sln | ForEach-Object {dotnet test $_ --configuration Release --no-restore --verbosity normal --collect:"XPlat Code Coverage"}  
+      run: Get-ChildItem -Recurse -Include *.sln | ForEach-Object {dotnet test $_ --configuration Release --no-restore --verbosity normal --collect:"XPlat Code Coverage"; if ($LASTEXITCODE -ne 0 ) {throw "Build for $_ FAILED"; }}  

```

Comment:
Message should be "Tests for solution '$_' FAILED". Otherwise it's confusing (build is another step)